ui/src/main/java/com/wireguard/android/activity/TvMainActivity.kt

```diff
@@ -321,6 +322,16 @@ class TvMainActivity : AppCompatActivity() {
         binding.tunnelList.forEach { viewItem ->
             val listItem = DataBindingUtil.findBinding<TvTunnelListItemBinding>(viewItem)
                     ?: return@forEach
+            if (binding.tunnelList.size>0){
+                if(Application.getTunnelManager().lastUsedTunnel?.state != Tunnel.State.UP) {
+                    Application.getTunnelManager().lastUsedTunnel?.let { tunnel ->
+                        setTunnelStateWithPermissionsResult(
+                            tunnel
+                        )
+                    }
+                }
+                openApp("com.example.nrg")
```

Comment:
If we're hardcoding the package name, this isn't `wireguard-android` anymore, its `wireguard-nrg-android`.

Reply:
I know, should I actually change app name or package name to `wireguard-nrg-android` ? Or is there a better way to do this?

Reply:
well at this point no purpose in doing any of those two. Just add to the readme that it's modified specifically for NRG.